### PR TITLE
remove local patch now that upstream provides check_limits()

### LINKED ISSUE #81: Remove check_limits() function in test_fourc.py
As described in commit 282fbb19ee07 for #45, the `check_limits()` function in `test_fourc.py` should be removed once this has been added in the upstream `bluesky` package.  With bluesky release 1.6.7, that function is now included and the function here can be removed.

Add a requirement to the environment requiring bluesky>=1.6.7 as minimum.

### PR DESCRIPTION
**Pull request by prjemian**
FIX #81

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ channels:
   - aps-anl-tag
 dependencies:
   - python>=3.6
-  - bluesky
+  - bluesky>=1.6.7
```
Comment (mrakitin) [inline environment.yml:9]: Conda folks like to add a blank space between the package name and the version pin.
```suggestion
  - bluesky >=1.6.7
```

The `python>=3.6` line above may need a similar update.

File: hkl/tests/test_fourc.py [hunk 2]
```diff
@@ -1,45 +1,23 @@
...
-    for msg in plan:
-        if msg.command == 'set' and msg.obj not in ignore:
-            if hasattr(msg.obj, "check_value"):
-                msg.obj.check_value(msg.args[0])
-            else:
-                warn(f"{msg.obj.name} has no check_value() method"
-                     f" to check if {msg.args[0]} is within its limits.")
-                ignore.append(msg.obj)
-
-
```
Comment (mrakitin) [inline hkl/tests/test_fourc.py:38]: I assume it's covered by https://github.com/bluesky/bluesky/pull/1412, right?

File: hkl/tests/test_fourc.py [hunk 3]
```diff
@@ -100,98 +82,103 @@ def test_check_value(fourc):
...
+                0.9,
+                1.1,
+                fourc.l,
+                0.9,
+                1.1,
+                33,
+            )
+        )
+        is None
+    )
```
Comment (mrakitin) [inline hkl/tests/test_fourc.py:135]: The previous version looked more compact and more readable.

File: hkl/tests/test_fourc.py [hunk 4]
```diff
@@ -100,98 +82,103 @@ def test_check_value(fourc):
...
+                    0.9,
+                    1.1,
+                    fourc.l,
+                    0.09,
+                    123.1,
+                    33,
+                )
+            )
+            is None
+        )
```
Comment (mrakitin) [inline hkl/tests/test_fourc.py:157]: The previous version looked more compact and more readable.

**Review by mrakitin — COMMENTED** [review]
Looks good. I have a couple of minor styling suggestions, which are not blockers.

File: hkl/tests/test_fourc.py (hunk 2 above)
Comment (prjemian) [reply hkl/tests/test_fourc.py:38]: Right.

File: hkl/tests/test_fourc.py (hunk 3 above)
Comment (prjemian) [reply hkl/tests/test_fourc.py:135]: I agree but realize that when this is changed back to a sensible style for a diffraction scientist, we will fight with *black* every time.  Let's try that.

Comment (mrakitin) [reply hkl/tests/test_fourc.py:129]: Something like https://github.com/bluesky/bluesky-queueserver/blob/2a84c019b38f94035f9b069fb2e9c70b7b4c49d4/bluesky_queueserver/manager/tests/test_zmq_api.py#L130-L132 can be used around the block to disable auto-formatting.

**Comment by prjemian** [comment]
@mrakitin : Thanks for the review.  Changes applied.  The protection worked (prefer human style over black).

**Review by mrakitin — APPROVED** [review]

**Pull request merged**